src/scripts/ci/travis.yml

```diff
@@ -34,6 +38,7 @@ env:
     - BUILD_MODE="nist"
     - BUILD_MODE="docs"
     - BUILD_MODE="lint"
+    - BUILD_MODE="docker-android"
```

Comment:
This should get moved up (maybe to just after the other cross builds). Travis does 5 builds in parallel, in the order specified here. If any of the final builds take a long time it ends with just 1 build running after everything else has completed, which lengthens the overall CI build time.